OpenTESArena/src/Game/Game.cpp

```diff
@@ -36,7 +34,8 @@ Game::Game()
 	this->options = OptionsParser::parse();
 
 	// Initialize virtual file system using the Arena path in the options file.
-	VFS::Manager::get().initialize(std::string(this->options->getArenaPath()));
+    std::string arena = File::toString(this->options->getArenaPath());
```

Comment:
I was thinking of doing the same thing, but upon closer inspection, I was a little unclear as to what the purpose of the `File` class was. Is it any type of file? Or a specific file for the engine?